backend/lib/richard_burton/auth_google.ex

```diff
@@ -0,0 +1,74 @@
+defmodule RichardBurton.Auth.Google do
+  @moduledoc """
+  Google idp implementation for RichardBurton.Auth behaviour
+  """
+  @behaviour RichardBurton.Auth
+
+  alias RichardBurton.User
+
+  @impl true
+  @spec init() :: {Map.t(), List.t()}
+  def init(), do: {get_config(), get_keys()}
+
+  @impl true
+  @spec verify(token :: String.t()) :: {:ok, String.t()} | :error
+  def verify(token) do
+    case Application.get_env(:richard_burton, :auth_config) do
+      {%{"issuer" => issuer}, keys} ->
+        verify(
```

Comment:
NIT: even though `verify/1` and `verify/4` are different functions, I think it's more idiomatic to name the private one something like `do_verify` so it's clearer for the reader which one is the public one.